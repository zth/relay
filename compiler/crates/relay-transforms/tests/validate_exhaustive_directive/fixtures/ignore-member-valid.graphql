fragment IgnoreMemberValid on User {
  nameRenderer @exhaustive(ignore: ["CustomNameRenderer"]) {
    ... on PlainUserNameRenderer { __typename }
    ... on MarkdownUserNameRenderer { __typename }
  }
}