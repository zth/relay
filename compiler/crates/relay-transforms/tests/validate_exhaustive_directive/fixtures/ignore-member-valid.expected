==================================== INPUT ====================================
fragment IgnoreMemberValid on User {
  nameRenderer @exhaustive(ignore: ["CustomNameRenderer"]) {
    ... on PlainUserNameRenderer { __typename }
    ... on MarkdownUserNameRenderer { __typename }
  }
}
==================================== OUTPUT ===================================
fragment IgnoreMemberValid on User {
  nameRenderer @exhaustive(ignore: ["CustomNameRenderer"]) {
    ... on PlainUserNameRenderer {
      __typename
    }
    ... on MarkdownUserNameRenderer {
      __typename
    }
  }
}
